zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/ElasticsearchSchemaManager.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.schema;
+
+import io.camunda.exporter.NoopExporterConfiguration.ElasticsearchConfig;
+import io.camunda.exporter.NoopExporterConfiguration.IndexSpecificSettings;
+import io.camunda.exporter.schema.descriptors.IndexDescriptor;
+import io.camunda.exporter.schema.descriptors.IndexTemplateDescriptor;
+import java.util.List;
+
+public class ElasticsearchSchemaManager implements SchemaManager {
+  private final SearchEngineClient elasticsearchClient;
+  private final List<IndexDescriptor> indexDescriptors;
+  private final List<IndexTemplateDescriptor> indexTemplateDescriptors;
+  private final ElasticsearchConfig elasticsearchConfig;
+
+  public ElasticsearchSchemaManager(
+      final SearchEngineClient elasticsearchClient,
+      final List<IndexDescriptor> indexDescriptors,
+      final List<IndexTemplateDescriptor> indexTemplateDescriptors,
+      final ElasticsearchConfig elasticsearchConfig) {
+    this.elasticsearchClient = elasticsearchClient;
+    this.indexDescriptors = indexDescriptors;
+    this.indexTemplateDescriptors = indexTemplateDescriptors;
+    this.elasticsearchConfig = elasticsearchConfig;
+  }
+
+  @Override
+  public void initialiseResources() {
+    indexTemplateDescriptors.forEach(this::createIndexTemplate);
```

Comment:
related to the comment above:
we have the configs available here. before this line we can create a component template using the settings from configs

Reply:
same point as above